glean_parser/lint.py

```diff
@@ -273,6 +274,18 @@ def check_expired_metric(
         yield ("Metric has expired. Please consider removing it.")
 
 
+PRIVATE_BUGTRACKER_DISALLOW_LIST = set(["mozilla-hub.atlassian.net"])
```

Comment:
I waffled a lot about allow vs. disallow list.  Thoughts?  I don't think actually checking the URLs at parse time is feasible -- for one, glean using products may be built in an offline environment...

Reply:
I think an allowlist might be better, as that allows us to encode a tighter policy (and I imagine, hypothetically, others using Glean might want to do the same). In the case of most Mozilla products, IMO the allowed URLs should be bugzilla.mozilla.org or github.

Reply:
I had considered that.  Github is a tricky one -- it could be public or private and I don't think there's any way to know from the URL.  I guess that problem exists whether its an allow or disallow list, though.